### ISSUE TITLE
Abstract criticism

### ISSUE THREAD

**Issue by dotsdl** [datreant/scipy-2016#4]
Once we have the long description settled (#2), I'll move on to sculpting the short description out of it. When we have a first draft we can use this thread as a focal point for criticism.

**Comment by dotsdl** [comment]
First draft of abstract just pushed (1979b1fcdd511bcbd22107c5339b9bc67eed7584). It's short enough, but does it make you want to attend the talk? See abstracts from last year for comparison: http://scipy2015.scipy.org/ehome/115969/292868/?&

**Comment by dotsdl** [comment]
Second draft: e765364f9ad5a4e7c6b6d0c0c11efac1adc3087f

**Comment by dotsdl** [comment]
@coredevs are we good to ship this? I have to submit by Friday.

**Comment by richardjgowers** [comment]
"vanilla by design" - I kind of get what you mean but I've also had non-native English speakers not know this idiom quite often

**Comment by dotsdl** [comment]
@richardjgowers I was conflicted about it, too, but a software audience will know what this means in this context, I think.

**Comment by richardjgowers** [comment]
Not sure you can start the second sentence with 'But'.. I tried rewriting it a little but I'm not sure it's actually an improvement

In science the filesystem often serves as a de facto database,
with the directory trees being the zeroth order data structure.
Working directly with heterogeneous datasets at this level
however is tedious and error prone.

**Comment by kain88-de** [comment]
Since this will appear online can you add links?

**Comment by dotsdl** [comment]
@richardjgowers I think starting with a conjunction is allowed, and it can be used to great effect to connect two clauses even if they are also broken into separate sentences. I'm not worried about it, but if you find an alternative opening with more bang then I'm all ears.

**Comment by dotsdl** [comment]
@kain88-de we can, yeah, but they'll be part of the wordcount. I'll add the main http://datreant.org site url. Luckily "datreant" is unique enough that it dominates the results when googling it.

**Comment by kain88-de** [comment]
As a non native speaker I'm also confused by the 'vanilla' design. e.g. for me having a 'vanilla' gnome3 experience means that I get just the raw gnome3 release without any extras. I would rephrase it like this.

> datreant is **modular and extendable** by design to allow specialized applications to be built on top of it, with MDSynthesis as an example for working with molecular dynamics simulation data.

And my take to remove the _but_. I don't think it is much better though.

> In science the filesystem often serves as a de facto database, with directory trees being the zeroth-order scientific data structure.  **Using the filesystem can be tedious and error prone when working with heterogeneous datasets**

**Comment by dotsdl** [comment]
@kain88-de I like **modular and extendable** over **vanilla**; I'll make the change.

I think the "But" should stay. For what it's worth, the google seems to say for ["starting a sentence with but"](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=starting%20a%20sentence%20with%20but) that it's perfectly fine, and often gives better writing when used well.

**Comment by kain88-de** [comment]
I would also leave it. After the _vanilla_ change I'm fine with the abstract.

**Comment by dotsdl** [comment]
Cool; pushed an update in 7c3e3e76b36196c42063aedc5a4c9fb8336fda1a (used "extensible" over "extendable", but that's really splitting hairs). I'll give the others time to chime in if they feel the need, and then I'll submit tomorrow. Thanks!

**Comment by dotsdl** [comment]
Added a bit in light of [changes in the long description](https://github.com/datreant/scipy-2016/issues/2#issuecomment-204088965) from conversation with @orbeckst. I think we're good to submit barring any objections.

**Comment by kain88-de** [comment]
It looks pretty good to me. I'm good with this one
